lms/static/js_test.yml

```diff
@@ -64,10 +65,12 @@ lib_paths:
 src_paths:
     - js
     - js/common_helpers
+    - teams/js
```

Comment:
@andy-armstrong and @dianakhuang here is the page--
https://openedx.atlassian.net/wiki/display/AC/Adding+a+feature+as+a+separate+Django+app